index.html

```diff
@@ -18,6 +18,14 @@
     </div>
     <nav>
       <img id="burgermenu" src="./img/burger.png" alt="">
+      <div id="menu">
+        <ul id="menu-container">
+          <li class="menu-item"><a class="menu-style" href="#">Portfolio</a></li>
+          <li class="menu-item"><a class="menu-style" href="#">About</a></li>
+          <li class="menu-item"><a class="menu-style" href="#">Contact</a></li>
+          <li class="menu-item"><img id="mail-icon" src="/img/icons/icons8-mail-48.png" alt=""></li>
+        </ul>
+      </div>
     </nav>
   </header>
   
```

Comment:
also, you forgot to include the illustrations in the headline section I suggest doing that 
![image](https://user-images.githubusercontent.com/61790587/165862990-f7cd9888-77a4-41a4-a711-edd808c313a4.png)